micromamba/src/activate.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019, QuantStack and Mamba Contributors
+//
+// Distributed under the terms of the BSD 3-Clause License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+
+#include "common_options.hpp"
+
+#include "mamba/api/configuration.hpp"
+
+#include "termcolor/termcolor.hpp"
+
+
+using namespace mamba;  // NOLINT(build/namespaces)
+
+void
+init_activate_parser(CLI::App* subcom)
+{
+    auto& config = Configuration::instance();
+
+    auto& env_name = config.insert(Configurable("activate_env_name", std::string(""))
```

Comment:
maybe here we could reuse an existing configurable instead of adding a new one (`prefix` or `env_name`)

Reply:
Sounds like a good idea to me, I just don't know how to do it.

Reply:
you can get an existing configurable with:
``` cpp
auto& env_name = config.at("env_name").get_wrapped<std::string>();
```
this syntax is required because of type erasure